packages/backend-core/src/db/couch/DatabaseImpl.ts

```diff
@@ -38,7 +39,10 @@ export function DatabaseWithConnection(
   if (!connection) {
     throw new Error("Must provide connection details")
   }
-  return new DatabaseImpl(dbName, opts, connection)
+  return new DDInstrumentedDatabase(
```

Comment:
We use these calls constantly. What is going to be the performance impact of this?

Reply:
I'm uncertain. If it's significant it will show  up in the metrics in DataDog and we can roll it back.

For what it's worth, Redis calls are also instrumented this way it's just that DataDog injects that for us at runtime.